lib/client.js

```diff
@@ -353,8 +326,10 @@ exports.createClient = function(reference, name) {
               err.body = res.body;
               err.code = res.body.code || 'UnknownError';
               err.statusCode = res.status;
-              if (reportStats) {
-                reportStats(false, 'http-' + res.status);
+              if (monitor) {
+                var d = process.hrtime(start);
+                monitor.measure(entry.name, d[0] * 1000 + d[1] / 1000000);
+                monitor.count(entry.name);
```

Comment:
Can we do: `monitor.measure([entry.name, 'server-error'], ...` or `monitor.measure([entry.name, 'client-error'], ...`

depending on whether it's a 4xx or 5xx error...